src/components/common/SearchInput/SearchInput.tsx

```diff
@@ -40,26 +64,67 @@ const SearchInput = React.forwardRef<SearchInputRef, SearchInputProps>(
     { onSearchClick, buttonTestId = 'store-search-button', ...props },
     ref
   ) {
+    const [searchQuery, setSearchQuery] = useState<string>('')
+    const [suggestionsOpen, setSuggestionsOpen] = useState<boolean>(false)
+    const [suggestions, setSuggestions] =
+      useState<SearchSuggestionsQueryQuery['search']['suggestions']>()
```

Comment:
The GraphQL generates a type for the Suggestion. What do you think about using it?!

Reply:
But this is the type generated from graphql

Reply:
Wdyt about this suggestion?
```tsx
useState<StoreSuggestions>()
```

This StoreSuggestions type is defined here: https://github.com/vtex-sites/base.store/blob/7fc7445aae8d71368c079bdd56dcb06dbb4612e6/%40generated/graphql/index.ts#L3539-L3542